tests/TestCase.php

```diff
@@ -68,6 +69,13 @@ public function callRawRequest(string $method, string $uri, $content, array $hea
         return $this->call($method, $uri, [], [], [], $server, $content);
     }
 
+    public function json($method, $uri, array $data = [], array $headers = [], ?VersionEnumContract $apiVersion = null): TestResponse
```

Comment:
let's keep this code in EmptyProject repository

Reply:
Done.